lib/domain/podcast_index/rss_podcast_index_chapters.dart

```diff
@@ -1,7 +1,14 @@
 import 'package:xml/xml.dart';
 
+/// Episodes that support the Podcasting 2.0 namespace can have a list of
```

Comment:
In keeping with Dart Doc practices, please add a blank line between the first sentence and whatever follows. And please adjust any other Dart Docs in this PR, accordingly.